File: Total amount of points.dart
int points(List<String> games) {
  int points = 0;
  for (var game in games) {
    var scores = game.split(":").map(int.parse).toList();
    int x = scores[0];
    int y = scores[1];
    points += (x > y) ? 3 : (x < y) ? 0 : 1;
  }
  return points;
}

File: Total amount of points/Total amount of points.ts
export function points(games : string[]): number {
  let array = games.map(v=>v.split(":"));
  let score = 0;
  for(let i=0;i<array.length;i++){
    let a = parseInt(array[i][0]);
    let b= parseInt(array[i][1]);
    if(a==b){score+=1}
    else if(a>b){score+=3}
    else{continue}
  }
  return score
}
